package.json

```diff
@@ -45,5 +51,7 @@
         "make test"
       ]
     }
+  },
+  "devDependencies": {
```

Comment:
I'm sort of puzzled why all webpack loaders are now required by eslint to be placed in the common dependencies instead of devDeps?